src/site/stages/build/drupal/process-lovell-pages.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable no-param-reassign */
+const cloneDeep = require('lodash/cloneDeep');
+
+const { camelize } = require('../../../utilities/stringHelpers');
+
+const LOVELL_TITLE_STRING = 'Lovell Federal';
+const LOVELL_FEDERAL_ENTITY_ID = '347';
+const LOVELL_TRICARE_ENTITY_ID = '1039';
+const LOVELL_VA_ENTITY_ID = '1040';
+const LOVELL_MENU_KEY = 'lovellFederalHealthCareFacilitySidebarQuery';
+
+function isLovellFederalPage(page) {
+  return (
+    page?.fieldAdministration?.entity?.entityId === LOVELL_FEDERAL_ENTITY_ID
+  );
+}
+
+function isLovellTricarePage(page) {
+  return (
+    page?.fieldAdministration?.entity?.entityId === LOVELL_TRICARE_ENTITY_ID
+  );
+}
+
+function isLovellVaPage(page) {
+  return page?.fieldAdministration?.entity?.entityId === LOVELL_VA_ENTITY_ID;
+}
+
+function getModifiedLovellPage(page, variant) {
+  const fieldOfficeMod = variant === 'tricare' ? 'Tricare' : 'VA';
+
+  // Modify the path
+  page.entityUrl.path = page.entityUrl.path.replace(
+    '/lovell-federal-health-care',
+    `/lovell-federal-${variant}-health-care`,
+  );
+
+  // Modify the title used for querying the menus
+  if (page.fieldOffice) {
+    page.fieldOffice.entity.entityLabel = page.fieldOffice.entity.entityLabel.replace(
+      LOVELL_TITLE_STRING,
+      `${LOVELL_TITLE_STRING} ${fieldOfficeMod}`,
+    );
+  }
+  if (page.fieldRegionPage) {
+    page.fieldRegionPage.entity.title = page.fieldRegionPage.entity.title.replace(
+      LOVELL_TITLE_STRING,
+      `${LOVELL_TITLE_STRING} ${fieldOfficeMod}`,
+    );
+  }
+
+  // Modify the title this will become more complex to handle specific cases
+  page.title = page.title.replace(
+    LOVELL_TITLE_STRING,
+    `${LOVELL_TITLE_STRING} ${fieldOfficeMod}`,
+  );
+
+  return page;
+}
+
+function lovellMenusModifyLinks(link) {
+  const { variant } = this;
+  const titleVar = variant === 'va' ? 'VA' : 'Tricare';
```

Comment:
This has been finalized to be all caps.
```suggestion
  const titleVar = variant === 'va' ? 'VA' : 'TRICARE';
```

Reply:
@swirtSJW I made this change and pushed up another commit.